src/main/java/fi/vm/sade/valinta/kooste/valintojentoteuttaminen/HakukohteenValintatiedot.java

```diff
@@ -1,12 +1,32 @@
 package fi.vm.sade.valinta.kooste.valintojentoteuttaminen;
 
+import java.util.Objects;
+
 /** Esittää eri palveluista koostettuja hakukohteen valintatietoja. */
 public class HakukohteenValintatiedot {
   public String hakukohdeOid;
   public Boolean hasValintakoe;
+  // laskettu osittain tai kokonaan
+  public Boolean laskettu;
```

Comment:
```suggestion
  public final Boolean laskettu;
```
ehkä muille, jos pysyvät muutumattomina